nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/es-audit-pageprovider-test-contrib.xml

```diff
@@ -3,97 +3,131 @@
 <component name="org.nuxeo.elasticsearch.PageProviderservice.test.contrib">
 
   <extension target="org.nuxeo.ecm.platform.query.api.PageProviderService"
-    point="providers">
+             point="providers">
```

Comment:
Our XML formatter is using 2 spaces as indent, AFAIK.

Reply:
i check my IDE configuration and i have indentation with 2 spaces. we can check together. fyi i tried reformatting an existing file [here](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/resources/elasticsearch-audit-index-test-contrib.xml#L5) and it didn't make any changes

Reply:
So I think IDEA is formatting differently XML files than Eclipse ...